src/HookFactory.php

```diff
@@ -129,6 +131,12 @@ public function generateHook($type, $programPath, $programName = null)
         // Use the program path if an alias/name is not given
         $programName = $programName ?: $programPath;
 
+        if ($multiple) {
+            $completionCommand = '${1} _completion';
```

Comment:
`{$1}` works under BASH, but doesn't resolve to anything under ZSH. Just using `$1` works correctly on both BASH and ZSH.